lld/ELF/Arch/Hexagon.cpp

```diff
@@ -60,17 +60,17 @@ Hexagon::Hexagon() {
 }
 
 uint32_t Hexagon::calcEFlags() const {
-  assert(!ctx.objectFiles.empty());
+  static const uint32_t DEFAULT_ARCH_REV = 0x60;
```

Comment:
The variable name does not follow  the convention. Since this variable is only used once, you can inline it and add a comment.

Reply:
Thanks for pointing it out.  Fixed.